src/main/java/winstone/HostConfiguration.java

```diff
@@ -55,7 +52,6 @@ public class HostConfiguration {
     private Map<String, String> args;
     private Map<String, WebAppContext> webapps;
     private ClassLoader commonLibCL;
-    private MimeTypes mimeTypes = new MimeTypes();
```

Comment:
Note that we are no longer replacing the default MIME types, but appending to them. As a result, we have deleted any MIME mappings that are already present in upstream Jetty and added a test to ensure that we are only adding to the existing MIME types rather than attempting to replace them.